numpy/distutils/msvc9compiler.py

```diff
@@ -0,0 +1,22 @@
+import os
+from distutils.msvccompiler import *
```

Comment:
It is best to avoid `import *`, instead either use the namespace, `distutils.msvccompiler.whatever`, which has the advantage of making it clear where everything comes from, or explicitly import those objects needed.